public/zt.html

```diff
@@ -53,7 +53,11 @@
     </div>
 
     <div class="container-sm status-content mt-5 p-5 rounded-4" id="ZT-form">
```

Comment:
Looks like symptom of html hiperDIVosis, if it's a form it should probably use \<form\> tag. 

Also maybe add some [bootstrap styling](https://getbootstrap.com/docs/5.3/forms/overview/#overview) to make this look better?

Reply:
That div is necessary to hold user selector and form together. I will change that tags name that it will be better suited to its job